content/docs/introduction/overview.md

```diff
@@ -17,11 +17,42 @@ of connected clients and billions of messages delivered daily.
 * each cloud component is individually scalable providing granular control
 * the cloud infrastructure is designed to be nearly unlimited in the size
 
+### Architecture
+
+This diagram illustrates the architecture of the Xmidt core routing cloud:
+
+![Xmidt core routing architecture](/assets/xmidt_core.png)
+
 ### Components
 
-### Architecture
+The Xmidt core routing cloud consists of several components.
+
+ * Talaria - secure web socket termination server
+ * Petasos - HTTP redirector for CPE devices
+ * Scytale - api service front end
+ * Caduceus - event delivery servicing agent
+ * Issuer - (optional) JWT issuer for authentication and authorization
+ * [Consul](https://www.consul.io/) - coordination service by HashiCorp
+ * [Prometheus](https://www.prometheus.io/) - (optional) metrics gathering
+ * [Trickster](https://github.com/Comcast/trickster) - (optional) metrics optimization
 
 ### When does it fit?
 
+Xmidt is designed to provide message delivery to **lots** of always on connected
```

Comment:
Not sure if we need to specify that the devices are always on if they should be continuously connected?

Reply:
I like that distinction better, as it's more clear.